src/pages/SetupPlot.vue

```diff
@@ -109,71 +105,65 @@ export default defineComponent({
       debounce(async (val) => {
         console.log(val)
         if (this.plotDirectory == this.defaultPath) return (this.validPath = true)
-        this.validPath = await util.native.dirExists(val)
+        this.validPath = await native.dirExists(val)
         if (this.validPath) await this.updateDriveStats()
+        return
       }, 500)
     )
   },
   computed: {
     chartData(): any {
-      return [this.utilizedGB, this.freeGB, this.allocatedGB]
+      return [this.stats.utilizedGB, this.stats.freeGB, this.allocatedGB]
     },
     printEstimatedTime(): string {
       return timeAgo.format(Date.now() + this.plotTimeHr).split("in")[1]
     },
     plotTimeHr(): number {
       return util.plotTimeMsEstimate(this.allocatedGB)
     },
-    totalDiskSizeGB(): number {
-      return parseFloat((this.driveStats.totalBytes / 1e9).toFixed(2))
-    },
-    diskPercentAvailable(): number {
-      return (this.safeAvailableGB / this.totalDiskSizeGB) * 100
-    },
-    diskPercentUsed(): number {
-      return 100 - this.diskPercentAvailable
-    },
-    allocatedGBChart(): number {
-      return (this.allocatedGB / this.safeAvailableGB) * 100
-    },
-    safeAvailableGB(): number {
-      return this.driveStats.freeBytes / 1e9
-    },
-    utilizedGB(): number {
-      return parseFloat((this.totalDiskSizeGB - this.safeAvailableGB).toFixed(2))
-    },
-    freeGB(): number {
-      return parseFloat((this.safeAvailableGB - this.allocatedGB).toFixed(2))
+    stats(): any {
```

Comment:
This and other functions in this file should return something meaningful instead of `any`.

This isn't serious if you have a very specific data structure here and writing `any` just to make compiler shut up.

Reply:
Sure I can type it